modules/ROOT/partials/misc/ui-languages.adoc

```diff
@@ -81,5 +86,6 @@ ifeval::[{forModuleLoaders} != true]
 | Thai (Thailand) | th_TH
 | Turkish | tr
 | Ukrainian | uk
+| Vietnamese | vi
 |===
-endif::[]
\ No newline at end of file
+endif::[]
```

Comment:
Not your fault, but looks like we need a `.editorconfig` to ensure line endings are consistent